Websites/webkit.org/demos/webxr-chess/chess.css

```diff
@@ -0,0 +1,157 @@
+
+html,
+body {
+    height: 100%;
+    background-color: white;
```

Comment:
Should we support dark mode?

Reply:
We can but it's not really about the text.

I also don't have a night version of the chess garden (although that would be neat for a future project).